apps/website/.vuepress/theme/styles/index.scss.rej

```diff
@@ -0,0 +1,8 @@
+diff a/apps/website/.vuepress/theme/styles/index.scss b/apps/website/.vuepress/theme/styles/index.scss	(rejected hunks)
+@@ -1,5 +1,6 @@
```

Comment:
Delete?

Reply:
not seeing these, will leave open for now and re-visit.

Reply:
Found and removed.